tests/framework/PostProcessors/ParetoFrontierPostProcessor/test_paretoFrontierInverted.xml

```diff
@@ -0,0 +1,60 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/PostProcessors/ParetoFrontierPostProcessor.ParetoFrontier</name>
```

Comment:
change ParetoFrontier to ParetoFrontierInverted

Reply:
edited